### PR TITLE
2647 financial data from csv

### PR DESCRIPTION
**Pull request by jacobvjk**
closes ADO 2647: https://dev.azure.com/2DegreesInvesting/2DegreesInvesting/_workitems/edit/2647

all change checks passed

### CODE REVIEW

**Review by MirijaHa — COMMENTED** [review]
- pls merge with master before approval
- we might want to simply drop those few rows that are not ASCII encoded as they should be useless any, or we could log their share as info to the user, but that would be entering unknown terrain. A conventinet way to detect non ASCII could be 
```
 dplyr::mutate(enc = 
stringi::stri_enc_mark(company_name))  %>% 
  dplyr::filter(enc != "ASCII") 
```

File: DESCRIPTION [hunk 1]
```diff
@@ -32,6 +32,7 @@ Imports:
     r2dii.plot,
     readr,
     readxl,
+    stringi,
```
Comment (MirijaHa) [inline DESCRIPTION:35]: please remove this unneeded dep, will approve then

**Review by MirijaHa — COMMENTED** [review]
1 last comemnt

File: DESCRIPTION (hunk 1 above)
Comment (jacobvjk) [reply DESCRIPTION]: done

**Review by MirijaHa — APPROVED** [review]

**Pull request merged**